src/cr.jl

```diff
@@ -160,9 +160,10 @@ kwargs_cr = (:M, :ldiv, :radius, :linesearch, :γ, :atol, :rtol, :itmax, :timema
       stats.niter = 0
       stats.solved, stats.inconsistent = true, false
       stats.timer = start_time |> ktimer
-      stats.status = "x = 0 is a zero-residual solution"
+      stats.status = "x is a zero-residual solution"
```

Comment:
```suggestion
      stats.status = "0 is a zero-curvature direction"
```

There should be a test after line 157:

```julia
if rNorm == 0
  stats.niter = 0
  stats.solved, stats.inconsistent = true, false
  stats.timer = start_time |> ktimer
  stats.status = "x = 0 is a zero-residual solution"
  history && push!(ArNorms, zero(T))
  solver.warm_start = false
  return solver
end
```

Reply:
Yes but:

1. The computation of `rNorm` and the test `rNorm == 0` should occur before `Ar` is computed;
2. the status when `ρ == 0` should be `b is a zero-curvature direction` (my fault).